snippets/nix.json

```diff
@@ -22,6 +22,11 @@
         "body": "hash = \"${1:sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=}\";",
         "description": "Empty hash"
     },
+    "Hash": {
```

Comment:
What's the functional difference between this `Hash` snippet and the one above it named `hash`? (Could we remove the other?)

Reply:
Hmm you are right, perhaps a better thing would be to add a snippet only for the content of this value. I pushed something.